frontends/web/src/routes/account/send/send.tsx

```diff
@@ -418,9 +398,10 @@ class Send extends Component<Props, State> {
     }
   };
 
-  private sendToSelf = (event: React.SyntheticEvent, recipientAddress: string) => {
-    this.setState({ recipientAddress });
-    this.handleFormChange(event);
+  private sendToSelf = (recipientAddress: string) => {
+    this.setState({ recipientAddress }, () => {
+      this.validateAndDisplayFee(true);
+    });
```

Comment:
`sendToSelf` only needs to set `recipientAddress` to be our own address like this. We previously called `handleFormChange` but I believe there's no point in doing so.

Reply:
very nice, 👍 

actually, the sendToSelf function is now the same as onReceiverAddressInputChange, I think you can remove sendToSelf and use onReceiverAddressInputChange.